chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -91,7 +91,8 @@ def self.bridge?(nic)
   def self.ovs_bridge?(nic)
     nic.kind_of?(::Nic::OvsBridge) or
       # If no ovs tools are installed there's no ovs switch
-      if ::Kernel.system("which ovs-vsctl > /dev/null 2>&1")
+      if ::Kernel.system("which ovs-vsctl > /dev/null 2>&1 && " \
+                         "systemctl -q is-active openvswitch.service")
```

Comment:
Well, if the service isn't active, surely it's totally pointless to run `ovs-vsctl` and cleaner to skip it rather than rely on it to return the exit code we want when the expected behaviour in that case might be undefined.